### DISCUSSION TITLE
How can I exclude files from checking?

### DISCUSSION THREAD

**Discussion by HannesTD** [valentjn/vscode-ltex#225] [Q&A] [answered]
Hi,

I am new to LTeX. 

Despite the great work you already put LTEX my team and I have some questions/suggestions.

Is it possible to exclude files from checking?

We do use some markdown files as internal lists to keep track of things. We don't want those files to be checked.

Best regards

Hannes

**Comment by valentjn** [comment, marked as answer]
You can use [magic comments](https://valentjn.github.io/vscode-ltex/docs/advanced-usage.html#magic-comments) to disable LT<sub>E</sub>X on specific files.

**Reply by HannesTD** [reply]
Thanks that helped a lot.
